blocks.py

```diff
@@ -0,0 +1,97 @@
+# Slack block kit - https://api.slack.com/block-kit
+
+from datetime import datetime
+
+# Get current date for time logging form
+def currentDate():
+    return datetime.now()
+
+# Time logging form blocks
+logForm = [
+    {
+        # Horizontal line
+        "type": "divider"
+    },
+    {
+        # Date picker
+        "type": "input",
+        "block_id": "date_input",
+        "element": {
+            "type": "datepicker",
+            # American format, YYYY:MM:DD
+            "initial_date": currentDate().strftime("%Y-%m-%d"),
```

Comment:
SQLite doesn't have a designated date object so the dates are stored as strings. This format allows them to be sorted because chronological order is the same as lexicographical order ("2022-05-07" > "1990-12-08" but "08-12-1990" > "07-05-2022"). I will make this more clear in the comment

Reply:
Fair enough, I was thinking this would change how the bot displayed the datepicker but having just tried it out it doesn't seem to make a difference so I don't think this is a problem :)